docs/user-guide/how_to_use_job_policy.md

```diff
@@ -0,0 +1,150 @@
+
+## Examples
+1. Set a pair of `event` and `action`.
+```yaml
+apiVersion: batch.volcano.sh/v1alpha1
+kind: Job
+metadata:
+  name: mpi-job
+spec:
+  minAvailable: 3
+  schedulerName: volcano
+  policies:
+    - event: PodEvicted   # job level policy. If any pod is evicted, restart the whole job.
+      action: RestartJob    
+  plugins:
+    ssh: [] 
+    svc: []
+  tasks:
+    - replicas: 1
+      name: mpimaster
+      template:
+        spec:
+          containers:
+            - command:
+                - /bin/bash
+                - -c
+                - |
+                  mkdir -p /var/run/sshd; /usr/sbin/sshd;
+                  MPI_HOST=`cat /etc/volcano/mpiworker.host | tr "\n" ","`;
+                  sleep 10;
+                  mpiexec --allow-run-as-root --host ${MPI_HOST} -np 2 --prefix /usr/local/openmpi-3.1.5 python /tmp/gpu-test.py;
+                  sleep 3600;
+              image: lyd911/mindspore-gpu-example:0.2.0
+              name: mpimaster
+              ports:
+                - containerPort: 22
+                  name: mpijob-port
+              workingDir: /home
+          restartPolicy: OnFailure
+    - replicas: 2
+      name: mpiworker
+      template:
+        spec:
+          containers:
+            - command:
+                - /bin/bash
+                - -c
+                - |
+                  mkdir -p /var/run/sshd; /usr/sbin/sshd -D; 
+              image: lyd911/mindspore-gpu-example:0.2.0
+              name: mpiworker
+              resources:
+                limits:
+                  nvidia.com/gpu: "1"
+              ports:
+                - containerPort: 22
+                  name: mpijob-port
+              workingDir: /home
+          restartPolicy: OnFailure
+```
+2. Set a pair of `events` and `action`.
+```yaml
+apiVersion: batch.volcano.sh/v1alpha1
+kind: Job
+metadata:
+  name: mpi-job
+spec:
+  minAvailable: 3
+  schedulerName: volcano 
+  plugins:
+    ssh: []
+    svc: []
+  tasks:
+    - replicas: 1
+      name: mpimaster
+      template:
+        spec: 
+          containers:
+            - command:
+                - /bin/bash
+                - -c
+                - |
+                  mkdir -p /var/run/sshd; /usr/sbin/sshd;
+                  MPI_HOST=`cat /etc/volcano/mpiworker.host | tr "\n" ","`;
+                  sleep 10;
+                  mpiexec --allow-run-as-root --host ${MPI_HOST} -np 2 --prefix /usr/local/openmpi-3.1.5 python /tmp/gpu-test.py;
+                  sleep 3600;
+              image: lyd911/mindspore-gpu-example:0.2.0
+              name: mpimaster
+              ports:
+                - containerPort: 22
+                  name: mpijob-port
+              workingDir: /home
+          restartPolicy: OnFailure
+    - replicas: 2
+      name: mpiworker
+      template:
+        spec:
+          policies:
+            - events: [PodEvicted, PodFailed]   # task level policy. If any pod is evicted or fails, restart the job.
```

Comment:
Can this yaml be parsed? I think it may be written as
```
- replicas: 2
  name: mpiworker
  policies:
    - events:
         - PodEvicted
         - PodFailed   # task level policy. If any pod is evicted or fails, restart the job.
      action: RestartJob
  template:
  ....
 ```

Reply:
I wonder it may be the typographical differences. It's ok in my local IDE. Let me try to update it.

Reply:
I've reviewed the display and it's also in github. Please refer to https://github.com/volcano-sh/volcano/blob/83ab5d000ae50e5678a630383105afc998b82ab7/docs/user-guide/how_to_use_job_policy.md

Reply:
l just tried applying the yaml, there are some errors.
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/22241503/164435226-75012ea3-04d6-4c68-b01e-5f755923df92.png">

Reply:
I'm sorry for take the policy at a wrong place. Let me fix it. Thanks!

Reply:
I've provided another 2 cases, which can be resued directly. Please take a review, thanks!